controllers/RatingController.js

```diff
@@ -0,0 +1,53 @@
+
+import models from '../database/models';
+import ServerResponse from '../modules';
+
+const { successResponse, notFoundError } = ServerResponse;
+
+/**
+ * @class RatingController
+ * @exports RatingController
+ */
+export default class RatingController {
+  /**
+     *
+     * @method getArticleRatings
+     *
+     * @description get all ratings for an article
+     *
+     * @param {object} req express request object
+     * @param {object} res express response object
+     * @param {object} next express next object
+     *
+     * @returns {object} returns profile info
+     * @memberof RatingController
+     */
+  static async getArticleRatings(req, res, next) {
+    try {
+      const { id } = req.params;
+      const isExist = await models.Rating.findOne({ where: { articleId: id } });
```

Comment:
This variable name `isExist` sounds off. Why not name it exactly what it is, `ratingExists` or something more clear?

Reply:
Okay